src/screens/InfoTodo.tsx

```diff
@@ -0,0 +1,54 @@
+import { StyleSheet, Text, View } from 'react-native'
+
+import { ClearButton } from '../components/ClearButton'
+import { InfoTodoNavProps } from '../navigation/types'
+import { Colors } from '../styles/colors'
+
+export default function InfoTodo({ navigation, route }: InfoTodoNavProps) {
+  const { id, title, description, checked } = route.params
+
+  function returnHomeScreen() {
+    navigation.navigate('HomeScreen', {
+      doneItem: {
+        id: id,
+      },
+    })
+  }
+
+  return (
+    <>
+      <View style={styles.view}>
+        <Text style={[styles.done, styles.text]}>
+          {checked ? 'isDone' : 'NotDone'}
```

Comment:
Check the Zeplin if these are the real strings. I know that for `is done` doesn't exist a copy, so you can try with these ones:

```suggestion
          {checked ? 'Done' : 'Not done'}
```